packages/registry/src/index.ts

```diff
@@ -2,47 +2,48 @@ import {
   type ActionParams,
```

Comment:
Large change set due to running formatter. Adding artblocks plugin was the only meaningful change.